src/ipc/StrandCoord.cc

```diff
@@ -37,20 +38,35 @@ void Ipc::StrandCoord::pack(TypedMsgHdr &hdrMsg) const
     hdrMsg.putString(tag);
 }
 
-Ipc::HereIamMessage::HereIamMessage(const StrandCoord &aStrand):
+Ipc::StrandMessage::StrandMessage(const MessageType msgType, const StrandCoord &aStrand):
+    messageType(msgType),
     strand(aStrand)
 {
 }
 
-Ipc::HereIamMessage::HereIamMessage(const TypedMsgHdr &hdrMsg)
+Ipc::StrandMessage::StrandMessage(const TypedMsgHdr &hdrMsg):
+    messageType(hdrMsg.type())
 {
     hdrMsg.checkType(mtRegistration);
```

Comment:
Forgotten leftovers from the old code?

```suggestion
    hdrMsg.checkType(hdrMsg.type());
```

Please `git grep` to double check that there are no other stale mtRegistration values in the PR branch. In fact, I would recommend renaming mtRegistration to mtStrandRegistration to eliminate such dangers for good, including during backporting!